packages/preload/src/index.ts

```diff
@@ -1490,6 +1490,18 @@ function initExposure(): void {
     return ipcInvoke('kubernetes-client:listIngresses');
   });
 
+  contextBridge.exposeInMainWorld('kubernetesWatchIngresses', async (): Promise<number> => {
```

Comment:
see other comment, should we make it more "generic" and do WatchInformer instead and pass in a type?

Reply:
I already have a generic `createInformer` function that accepts a type as a param, i can expose that function